src/bsoncxx/oid.cpp

```diff
@@ -23,12 +23,7 @@
 namespace bsoncxx {
 BSONCXX_INLINE_NAMESPACE_BEGIN
 
-const oid::init_tag_t oid::init_tag{};
-
-oid::oid() : _is_valid(false) {
-}
-
-oid::oid(init_tag_t) : _is_valid(true) {
+oid::oid() : _is_valid(true) {
```

Comment:
In that case, what should I do with operator bool()? Should I have it always return true and deprecate it or just get rid of it entirely?